apps/dialtone-documentation/docs/components/notice.md

```diff
@@ -32,21 +32,311 @@ Used in most scenarios when the message should be noticeable but not dominate.
+          </div>
+          <svg>...</svg>
+        </span>
+      </span>
+    </button>
+  </div>
+</aside>
+<aside class="d-notice d-notice--warning">
+  <div aria-hidden="true" class="d-notice__icon">
+    <span class="d-icon__wrapper">
+      <div aria-busy="true" role="status" aria-label="" class="d-icon d-icon--size-400" style="display: none;">
+        <div
+          class="skeleton-placeholder d-bar-circle skeleton-placeholder--animate"
+          style="animation-delay: 0ms; animation-duration: 1000ms; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
+        ></div>
+      </div>
+      <svg>...</svg>
+    </span>
+  </div>
+  <div class="d-notice__content" role="status">
+    <p class="d-notice__title">Warning title (optional)</p>
+    <p class="d-notice__message">
+      <span> Message body with <a href="#" class="d-link d-link--muted">a link</a>. </span>
+    </p>
+  </div>
+  <div class="d-notice__actions">
+    <button type="button" class="base-button__button d-btn d-btn--outlined d-btn--muted d-btn--sm">
+      <span class="d-btn__label base-button__label"> Action </span>
+    </button>
+    <button type="button" aria-label="Close" class="base-button__button d-btn d-btn--sm d-btn--circle d-btn--icon-only">
+      <span class="base-button__icon d-btn__icon d-btn__icon--left">
+        <span class="d-icon__wrapper">
+          <div aria-busy="true" role="status" aria-label="" class="d-icon d-icon--size-200" style="display: none;">
+            <div
+              class="skeleton-placeholder d-bar-circle skeleton-placeholder--animate"
+              style="animation-delay: 0ms; animation-duration: 1000ms; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
+            ></div>
+          </div>
+          <svg>...</svg>
+        </span>
+      </span>
+    </button>
+  </div>
+</aside>
+'
+vueCode='
+<dt-notice
+  title="Base title (optional)"
+>
+  <span>
+    Message body with
+    <a
+      href="#"
+      class="d-link"
+      :class="linkClass"
+    >a link</a>.
+  </span>
+  <template #action>
+    <dt-button
+      size="sm"
+      importance="outlined"
+      :kind="buttonKind"
+      @click="$attrs.onClick"
+    >
+      Action
+    </dt-button>
+  </template>
+</dt-notice>
+<dt-notice
+  title="Info title (optional)"
+  kind="info"
+>
+  <span>
+    Message body with
+    <a
+      href="#"
+      class="d-link"
+      :class="linkClass"
+    >a link</a>.
+  </span>
+  <template #action>
+    <dt-button
+      size="sm"
+      importance="outlined"
+      :kind="buttonKind"
+      @click="$attrs.onClick"
+    >
+      Action
+    </dt-button>
+  </template>
+</dt-notice>
+<dt-notice
+  title="Error title (optional)"
+  kind="error"
+>
+  <span>
+    Message body with
+    <a
+      href="#"
+      class="d-link"
+      :class="linkClass"
+    >a link</a>.
+  </span>
+  <template #action>
+    <dt-button
+      size="sm"
+      importance="outlined"
+      :kind="buttonKind"
```

Comment:
set to explicit kind instead of "buttonKind" the user won't know what buttonKind is